drivers/firmware/scmi/Kconfig

```diff
@@ -64,6 +64,12 @@ config ARM_SCMI_TRANSPORT_INIT_PRIORITY
 	help
 	  SCMI transport driver device initialization priority.
 
+config SCMI_CHAN_SEM_TIMEOUT_USEC
+	int "SCMI channel semaphore timeout(unit: usec)"
+	default 500
+	help
+	  SCMI channel semaphore timeout(unit: usec).
```

Comment:
how about something more descriptive like: `Maximum number of microseconds to wait for a reply from the platform`?